integration/kubernetes/init.sh

```diff
@@ -30,8 +30,9 @@ wait_pods_ready()
         local controller_pod="kube-controller-manager.*1/1.*Running"
         local etcd_pod="etcd.*1/1.*Running"
         local scheduler_pod="kube-scheduler.*1/1.*Running"
+	local dns_pod="coredns.*1/1.*Running"
 
-        local system_pod=($apiserver_pod $controller_pod $etcd_pod $scheduler_pod)
+        local system_pod=($apiserver_pod $controller_pod $etcd_pod $scheduler_pod $dns_pod)
```

Comment:
can you please convert all leading spaces inside this function to tabs?

Reply:
thanks. I'll update asap. ;)